CHANGELOG.md

```diff
@@ -1,6 +1,61 @@
 # Changelog
 <!-- Autogenerated with go run tools/releases/changelog.go changelog.md -->
 
+## [1.8.3]
+> Released on 2023/01/12
+
+* chore(deps): bump helm.sh/helm/v3 from 3.8.1 to 3.10.3 [#5598](https://github.com/kumahq/kuma/pull/5598) @mergify
+* chore(deps): update coreDNS to 1.10.0 (backport #5626) [#5656](https://github.com/kumahq/kuma/pull/5656) @mergify
+* chore: remove Apache license header from generated files (backport #5565) [#5617](https://github.com/kumahq/kuma/pull/5617) @mergify

```

Comment:
I think we probably only want `chore(deps)` to be in the changelog right?

Reply:
Not sure. This likely shouldn't be in the changelog but the `chore(helm)` in 1.7.4 makes sense no?

Reply:
yeah that should be there but then it probably should be `fix`

Reply:
Do you mean that we should only allow chore for deps?

Here's a list of the last usages of chore:

```
➜  kuma git:(refactorEnv) git log --pretty=oneline | grep chore | grep -v 'chore(deps)' | head -20
1ec21aa13d7087af03d677d73290532d0cf78b14 chore: strip paths from stacktraces (#5701)
17fff0a5c6dd6b0aab5bd6ca22d850788d4f6b1a chore(MeshHTTPRoute): use correct return type of GetDefault (#5697)
6195e4bf2345b328e4e220589ab1a5a6da3707fa chore: run CodeQL only on master (#5678)
78ad0dc4f4cab0d2d26dbda24b7733c8f317b165 chore: fix README.md syntax errors (#5665)
e6552b6132f4be3d1356f24f7f2f12c406bfbb3c chore: validation error path merging with index/keys (#5652)
6f45254d7048e734317c723d433b94bcd406f02a chore: fix merge error in store options (#5637)
5960922f84b5c3413932f417a6aaabfd07e02b0e chore: remove unused proto import (#5605)
13a381aa990ae3a488803da5b84b356fc3874bf9 chore: do not hide errors when generating policy (#5573)
fdac0e281c90aac81fa0413d69eaf4824a171fea chore: remove Apache license header from generated files (#5565)
36e51a890e2943ad9442f4d67649646b37cabafd chore: unify delete mesh resources (#5529)
6f842a5af1a308651551358199eac16339e52f39 chore(kumactl): remove default `--valid-for` for tokens (#5521)
3727b6236be4127e39efbdfd1e2c3709c83e0bd1 chore(e2e): switch to new transparent proxy engine (#5485)
2d627d21eabf2f670b0111d0d39c14d0ef9a2e6f chore(e2e): cleanup gateway client which is not in mesh (#5514)
b13139a79d4d8ebb617eccbdcd852fefd233989b chore: graduate debug-fast target (#5502)
b126bc8aca5f555696fb8ad7eb92074b8aac6f6a chore(k8s): always inject Kuma as the first container (#5436)
b7e0339d6a7a47446765ef527e1cdfa5f2791ecd chore(helm): use dummy chart versions (#5447)
fc5db360f3df74fe8e6b933cc4ab60dc7fad1213 chore(docs): fix madr numbering (#5449)
eca78f321dfe41b724a1e004db801721a7c00f3a chore(helm): fix logo URL (#5444)
2b4fab641f5551c7c0a2af0bd0ce17a1d5052395 chore(ebpf): bump kuma-net to v0.8.9 (#5366)
8198e762f5c8ea664047bb9caf57ab214376be9a chore: fetch Envoy from Github artifacts (#5329)
```

Seems like they are not all valid but some of these do apply for being chore

Reply:
I mean that `chore` should, with the exception of `chore(deps)`, not be used for changes relevant to users. What is `chore` possibly supposed to mean?

Reply:
Ok so IIUC what you mean. The changelog tool should not generate lines for any chore except `chore(deps)`. Is that correct? 

In that case there's a few small fixes needed here for chore being used for actual important code changes

Reply:
Script improved: https://github.com/kumahq/kuma/pull/5728